CHANGELOG.md

```diff
@@ -1,7 +1,147 @@
+- Enable views to be nested within **Text** component (#23195) ([a2285b1](https://github.com/facebook/react-native/commit/a2285b1) by [@rigdern](https://github.com/rigdern))
+- Add support for cancelling fetch requests with **AbortController** ([5e36b0c](https://github.com/facebook/react-native/commit/5e36b0c) by [@janicduplessis](https://github.com/janicduplessis))
+- Add TypeScript support to the ESLint config ([b49fdd1](https://github.com/facebook/react-native/commit/b49fdd1) by [@satya164](https://github.com/satya164))
+- Add `.eslintrc` file to generated template. ([395197d](https://github.com/facebook/react-native/commit/395197d) by [@michalchudziak](https://github.com/michalchudziak))
+- Add CLI bin to package.json ([ab1edae](https://github.com/facebook/react-native/commit/ab1edae) by [@Esemesek](https://github.com/Esemesek))
+- Update react-native in `template/package.json` ([6df2ede](https://github.com/facebook/react-native/commit/6df2ede) by [@Esemesek](https://github.com/Esemesek))
+- Make Jest transform react-native-community/ packages by default ([7e23c7c](https://github.com/facebook/react-native/commit/7e23c7c) by [@thymikee](https://github.com/thymikee))
+- Add support for native driven string interpolation in **Animated** ([5e4a589](https://github.com/facebook/react-native/commit/5e4a589) by [@msand](https://github.com/msand))
+
+#### Android specific
+
+- Add schema for **AndroidSwipeRefreshLayoutSchema** ([9202d4f](https://github.com/facebook/react-native/commit/9202d4f) by [@mdvacca](https://github.com/mdvacca))
+- Add support for **custom fonts** ([f01c4e2](https://github.com/facebook/react-native/commit/f01c4e2) by [@dulmandakh](https://github.com/dulmandakh))
+- Add ram bundle to android template ([4478e50](https://github.com/facebook/react-native/commit/4478e50) by [@gengjiawen](https://github.com/gengjiawen))
+- Add public method `setCustomClientBuilder` to the native `com.facebook.react.modules.network.NetworkingModule`, that allows customizing the OkHttpClient per-request for greater control over HTTP requests ([81fcaa1](https://github.com/facebook/react-native/commit/81fcaa1))
+- Expose `collapsable` as React Prop in **LayoutShadowNode** ([58c16e6](https://github.com/facebook/react-native/commit/58c16e6) by [@mdvacca](https://github.com/mdvacca))
+- Implement data detection for android **Text** elements (#19216) ([b1251d0](https://github.com/facebook/react-native/commit/b1251d0) by [@kpink224](https://github.com/kpink224))
+
+#### iOS specific
+
+- **MountingManager** can create views with props in one step instead of two ([bbd925c](https://github.com/facebook/react-native/commit/bbd925c) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Fire timers in the background via **NSTimer** (#23674) ([97c414e](https://github.com/facebook/react-native/commit/97c414e) by [@cojo](https://github.com/cojo))
```

Comment:
FYI, background not really can works, needs fixed in https://github.com/facebook/react-native/pull/24649.